pkg/comp-functions/functions/common/resources_test.go

```diff
@@ -0,0 +1,110 @@
+package common
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	vshnv1 "github.com/vshn/appcat/v4/apis/vshn/v1"
+	"github.com/vshn/appcat/v4/pkg/common/utils"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/functions/commontest"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func TestResources(t *testing.T) {
+
+	ctx := context.Background()
+	svc := commontest.LoadRuntimeFromFile(t, "plans.yaml")
+	resources, err := utils.FetchPlansFromConfig(ctx, svc, svc.Config.Data["defaultPlan"])
```

Comment:
change the variable please, it was difficult to figure out it's the plan resource variable. Like `defaultPlan`. Also add a comment here or to each unit test what are the default plan's values, like memory, cpu. It will help to not check the file each time :)

Reply:
Done